smallrye-reactive-messaging-provider/src/test/java/io/smallrye/reactive/messaging/inject/EmitterInjectionTest.java

```diff
@@ -268,13 +274,15 @@ public boolean isCaught() {
         }
 
         public void run() {
-            emitter.send("a").send("b");
+            emitter.send("a");
```

Comment:
The tests need to check that the returned CompletionStage is completed when the message is processed.

Reply:
added more tests